lib/DDG/Spice/NobelPrize.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::NobelPrize;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use Data::Dumper;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.nobelprize.org/v1/prize.json?' 
+                . '&year=$1' 
+                . '&category=$2';
+
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers startend => 'nobel';
```

Comment:
> Refining the trigger to only certain kinds of phrases, is that necessary? 
> I mean i get that people can try varying strings but we can safely ignore the complications and still show the relevant data. Don't you think?

Why bother analyzing **_every_** query that contains `nobel` when we know only a small subset is relevant to us?

If we make it an `any` trigger with the more specific set we should be able to trigger on only the most relevant queries to us.

`triggers any => "nobel prize", "nobel peace prize" ...` would trigger on `1996 nobel peace prize winner`

The best way to see what's popular is to think about ways people would ask and check the autocomplete result.

![drake_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15790629/539744c4-29a0-11e6-92fe-55aaee288dbe.png)

At a glance, all of these would trigger and none are relevant, which means we'd be throwing away the majority of queries triggering this. We already know that we can use more specific triggers so we might as well.